Failure to find org.btools:brouter-mem-router

I'm trying to build master branch using the README command: `mvn clean install -Dandroid.sdk.path=`.

However it fails with:

> [ERROR] Failed to execute goal on project brouter-server: Could not resolve dependencies for project org.btools:brouter-server:jar:1.4.11: Failure to find org.btools:brouter-mem-router:jar:1.4.11 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :brouter-server

--- a/brouter-server/pom.xml
+++ b/brouter-server/pom.xml
@@ -51,11 +51,6 @@
             <artifactId>brouter-map-creator</artifactId>
             <version>${project.version}</version>
 		</dependency>
-        <dependency>
-            <groupId>org.btools</groupId>
-            <artifactId>brouter-mem-router</artifactId>
-            <version>${project.version}</version>
-        </dependency>
 		<dependency>
 			<groupId>junit</groupId>
 			<artifactId>junit</artifactId>

